src/routing/routeBuilderContext.ts

```diff
@@ -57,6 +57,11 @@ export default class RouteBuilderContext {
         return this;
     }
 
+    removeQueryParams() {
```

Comment:
We already have this function, 10 rows below we have function clear() that does this exact thing. The functions of this class that modifies the params have the names: append(), remove(), set() and clear(). The name removeQueryParams does not fit in.
The current names are maybe not clear enough, and we could probably change them to something like: addQueryParam(), removeQueryParam(), setQueryParam(), clearQueryParams()

Reply:
I know removeQueryParams doesn't fit in but I didn't want to do refactoring since I would have to change everything... Should've added TODO comments.

Reply:
Using clear() - didn't see it back then...